apps-rendering/src/components/layout/standard.stories.tsx

```diff
@@ -1,24 +1,47 @@
 // ----- Imports ----- //
-import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDisplay } from '@guardian/libs';
 import { breakpoints } from '@guardian/source-foundations';
-import { partition } from '@guardian/types';
+import type { Option } from '@guardian/types';
+import { OptionKind, partition, some } from '@guardian/types';
+import Comment from 'components/layout/comment';
 import Standard from 'components/layout/standard';
-import { article, matchReport, review } from 'fixtures/item';
+import {
+	analysis,
+	article,
+	comment,
+	editorial,
+	letter,
+	matchReport,
+	review,
+} from 'fixtures/item';
+import { deadBlog, live } from 'fixtures/live';
+import type { Item } from 'item';
+import type { ReactElement } from 'react';
 import { renderAll } from 'renderer';
+import Live from './live';
 
-// ----- Stories ----- //
+// ----- Functions ----- //
+
+const formatFromItem = (
+	item: Item,
+	forceDisplay: Option<ArticleDisplay>,
+): ArticleFormat => ({
+	theme: item.theme,
+	design: item.design,
+	display:
+		forceDisplay.kind === OptionKind.Some
+			? forceDisplay.value
+			: item.display,
```

Comment:
Possible alternative, up to you which you prefer.

```suggestion
	display: withDefault(item.display)(forceDisplay),
```

Reply:
Oh yes! Forgot about that. Much neater